.eslintignore

```diff
@@ -1,5 +1,3 @@
 coverage/
 dist/
 node_modules/
-# We just want to lint the very top level
-packages/
```

Comment:
Removed this because it was causing ESLInt to completely break for me in all subpackages and I'm not sure why! I've replaced this with updating the root `lint` call to only lint `__tests__` since thats the only directory in the root level that has code that needs linting.